bot/cogs/animals.py

```diff
@@ -0,0 +1,122 @@
+"""Commands for the animal module."""
+import discord
+import aiohttp
+
+from discord.ext import commands
```

Comment:
you should import what you need `from discord.ext.commands`
```suggestion
from discord.ext.commands import Cog, command, bot
```
then change the rest of the file to match this change